collect_app/src/main/res/layout/url_widget_answer.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/url_button"
+        style="@style/Widget.Collect.Button.Custom"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/margin_small"
+        android:text="@string/open_url"
+        android:layout_marginStart="@dimen/margin_standard"
+        android:layout_marginEnd="@dimen/margin_standard"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        android:layout_marginBottom="@dimen/margin_extra_small"
```

Comment:
Just a tiny change while we're here. Often it's nicer to have the margin between two items be on the second item (the `MaterialTextView`) rather than the first (the 1MaterialButton`) here. This often helps when the first/last items have different margins and also makes it easier when deleting items. I'd probably move the `marginBottom` here so it's a `marginTop` on the `MaterialTextView`.

Reply:
Ohh! Thanks for the advice. I will remember that.